packages/back-end/test/api/features.test.ts

```diff
@@ -36,17 +36,22 @@ describe("features API", () => {
   it("can create new features", async () => {
     setReqContext({
       org,
+      models: {
+        safeRollout: {
+          getAllPayloadSafeRollouts: jest.fn().mockResolvedValue(new Map()),
+        },
+      },
       permissions: {
         canPublishFeature: () => true,
         canCreateFeature: () => true,
       },
       getProjects: async () => [{ id: "project" }],
     });
 
-    createFeature.mockImplementation((v) => v);
-    getFeature.mockReturnValue(undefined);
-    addTags.mockReturnValue(undefined);
-    createInterfaceEnvSettingsFromApiEnvSettings.mockReturnValue(
+    (createFeature as jest.Mock).mockImplementation((v) => v);
+    (getFeature as jest.Mock).mockReturnValue(undefined);
+    (addTags as jest.Mock).mockReturnValue(undefined);
+    (createInterfaceEnvSettingsFromApiEnvSettings as jest.Mock).mockReturnValue(
       "createInterfaceEnvSettingsFromApiEnvSettings"
     );
     getSavedGroupMap.mockReturnValue("savedGroupMap");
```

Comment:
Should we cast these 2 as `jest.Mock` too? (`getSavedGroupMap` & `getApiFeatureObj`)